styles/styles.css

```diff
@@ -0,0 +1,39 @@
+@import url('https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@600&family=Press+Start+2P&display=swap');
+
+* {
+    font-family: 'Cormorant Garamond', serif;
+}
+h1 {
+    font-size: 4.5rem;
+}
+
+h1, h2, h3, h4, p {
+    padding-left: 25%;
+}
+
+h2 {
+    font-size: 3rem;
+}
+h3 {
+    font-size: 2.5rem;
+}
+
+h4 {
+    font-size: 2rem;
+}
+
+p {
+    font-size: 2rem;
+    max-width: 50%;
+}
+
+@media screen and (max-width:800px) {
+    h1, h2, h3, h4, p {
+        padding-left: 5%;
+    }
+    p {
+        max-width: 100%;
+        padding-right: 5%;
```

Comment:
again here you have some slightly mysterious percent padding, not sure what the intention is?

Reply:
all 'padding' issues sorted - also allowed me to remove the media query, as it is responsive by default!